cvmfs/mountpoint.cc

```diff
@@ -1460,44 +1460,42 @@ bool MountPoint::CreateDownloadManagers() {
       options_mgr_->IsOn(optarg)) {
     download_mgr_->EnableHTTPTracing();
     if (options_mgr_->GetValue("CVMFS_HTTP_TRACING_HEADERS", &optarg)) {
-      std::vector<std::string> tokens = SplitString(optarg, '|');
-      sanitizer::AlphaNumSanitizer sanitizer;
-
-      for (unsigned int i = 0; i < tokens.size(); i++) {
-        std::string token = Trim(tokens[i]);
+      if (optarg.size() > 200) {
+        LogCvmfs(kLogCvmfs, kLogSyslogErr | kLogDebug,
+             "CVMFS_HTTP_TRACING_HEADERS too large ( max 200 chars, given %d )",
+             optarg.size());
```

Comment:
If we look at the entire CVMFS_HTTP_TRACING_HEADERS, I think we should go higher, let's say 1000 characters.